src/modals/createPageOrRecordModal.ts

```diff
@@ -139,8 +140,7 @@ export async function createPageOrRecordModal(
             actionId: NotionPageOrRecord.TITLE_ACTION,
         }
     );
-
-    blocks.push(titleOfPageOrRecordBlock);
+    if(page || parent) blocks.push(titleOfPageOrRecordBlock);
```

Comment:
```suggestion

    if(page || parent) blocks.push(titleOfPageOrRecordBlock);
```

Add space above for readability